### DISCUSSION TITLE
Matlab modules

### DISCUSSION THREAD

**Discussion by wharfbanger** [alexisInSAR/EZ-InSAR#97] [Help]
Dear All, I am very interested in trialing EZ-InSAR.  I already use StaMPS in Ubuntu (20.04), with 4 MATLAB modules:

Image Processing Toolbox 
Parallel Computing Toolbox 
Signal Processing Toolbox 
Statistics and Machine Learning Toolbox 

I note that EZ-InSAR also requires:

Curve Fitting Toolbox 
Financial Toolbox 
Mapping Toolbox 
Optimization Toolbox

Are these 4 additional MATLAB modules absolutely necessary?  They will be very costly to purchase!

**Comment by alexisInSAR** [comment]
Yes and no.

Most of the modules are required for StaMPS, but Mapping is used by EZ-InSAR to create the maps. You can ignore these visualisations and just use MintPy.

**Comment by wharfbanger** [comment]
Thanks Alexis,I already have StaMPS installed, so already had to purchase 4 MATLAB modules. The other 4 I don’t have.  Perhaps I can install and run EZ-InSAR with just the 4 core MATLAB modules (but without visualisations as you say)?Thanks,MarkSent from my iPhoneOn 17/05/2024, at 10:39 AM, Alexis Hrysiewicz ***@***.***> wrote:﻿
Yes and no.
Most of the modules are required for StaMPS, but Mapping is used by EZ-InSAR to create the maps. You can ignore these visualisations and just use MintPy.
